src/coreclr/scripts/jitutil.py

```diff
@@ -50,7 +50,16 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_val, exc_tb):
         os.chdir(self.cwd)
         if not self._skip_cleanup:
-            shutil.rmtree(self.mydir)
+            try:
+                shutil.rmtree(self.mydir)
+            except Exception as ex:
+                logging.warning("Warning: failed to remove directory \"%s\": %s", self.mydir, ex)
+                # Print out all the remaining files and directories.
+                for dirpath, dirnames, filenames in os.walk(self.mydir):
```

Comment:
Is there any point in printing this? What if `os.walk` fails for that directory as well?

Reply:
Yes, that will be good because I have seen things fail in APIs around files/folders. We can't do much about it except to catch those exceptions and move forward.